src/module_1/module_1_meteo_api.py

```diff
@@ -70,16 +84,25 @@ def get_data_meteo_api(city, from_year, until_year):
     print("You reached the number of maximum attempts. Stopping the execution")
     sys.exit()
 
+
 def clean_data(raw_data):
+    """
+    (Not used)
+    Function defined for cleaning None values inside a dictionary
+    """
     cleaned_data = {}
     for key, value in raw_data.items():
```

Comment:
This assumes that `raw_data` is a `dict`, either verify it or use a type hint in the signature of the function